src/Illuminate/Support/ValidatedInput.php

```diff
@@ -100,6 +100,36 @@ public function all()
         return $this->input;
     }
 
+    /**
+     * Check if the validated input contains one or many keys.
+     *
+     * @param  mixed  $keys
+     * @return bool
+     */
+    public function has($keys)
+    {
+        $keys = is_array($keys) ? $keys : func_get_args();
+
+        foreach ($keys as $key) {
+            if (! array_key_exists($key, $this->input)) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    /**
+     * Check if the validated input does not contain one or many keys.
+     *
+     * @param  mixed  $keys
+     * @return bool
+     */
+    public function doesntHave($keys)
```

Comment:
I feel like `missing` is more Laravel-ish

Reply:
I had the same thought, `missing()` is used as the method name when interacting with a plain `request()` instance.

https://github.com/laravel/framework/blob/9.x/src/Illuminate/Http/Concerns/InteractsWithInput.php#L220

Reply:
I'll get that updated 👍🏼